contracts/token/ERC20RebasableBridged.sol

```diff
@@ -60,17 +63,20 @@ contract ERC20RebasableBridged is IERC20, IERC20Wrapper, IBridgeWrapper, ERC20Me
     /// @param decimals_ The decimals places of the token
     /// @param tokenToWrapFrom_ address of the ERC20 token to wrap
     /// @param tokenRateOracle_ address of oracle that returns tokens rate
+    /// @param tokenRateOracleDecimals_ Decimals of the oracle response.
     /// @param l2ERC20TokenBridge_ The bridge address which allows to mint/burn tokens
     constructor(
         string memory name_,
         string memory symbol_,
         uint8 decimals_,
         address tokenToWrapFrom_,
         address tokenRateOracle_,
+        uint8 tokenRateOracleDecimals_,
         address l2ERC20TokenBridge_
     ) ERC20Metadata(name_, symbol_, decimals_) {
         TOKEN_TO_WRAP_FROM = IERC20(tokenToWrapFrom_);
         TOKEN_RATE_ORACLE = ITokenRateOracle(tokenRateOracle_);
+        TOKEN_RATE_ORACLE_DECIMALS = tokenRateOracleDecimals_;
```

Comment:
:warning:  Why can't we retrieve the value from Oracle and initialize the immutable here without the need to pass it through the constructor?

What if we passed a different value in the end breaking the setup?